rootston/keyboard.c

```diff
@@ -116,86 +155,64 @@ static bool keyboard_keysym_press(struct roots_keyboard *keyboard,
 	return false;
 }
 
-static void keyboard_keysym_release(struct roots_keyboard *keyboard,
-		xkb_keysym_t keysym) {
-	ssize_t i = keyboard_pressed_keysym_index(keyboard, keysym);
-	if (i >= 0) {
-		keyboard->pressed_keysyms[i] = XKB_KEY_NoSymbol;
-	}
-}
 /*
- * Process keypresses from the keyboard as xkb sees them.
+ * Get keysyms and modifiers from the keyboard as xkb sees them.
  *
  * This uses the xkb keysyms translation based on pressed modifiers and clears
  * the consumed modifiers from the list of modifiers passed to keybind
  * detection.
  *
- * (On US layout) this will trigger: [Alt]+[at]
+ * On US layout, this will trigger: Alt + @
```

Comment:
I don't understand this comment line.

Reply:
Oh I see, the context for this comment comes from several lines below. This is kind confusing to read and should probably be made self contained for each.